src/RestApi/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -187,19 +192,23 @@ public function get_item_response( $cart_item ) {
 		$product = $cart_item['data'];
 
 		return [
-			'key'               => $cart_item['key'],
-			'id'                => $product->get_id(),
-			'quantity'          => wc_stock_amount( $cart_item['quantity'] ),
-			'name'              => $product->get_title(),
-			'sku'               => $product->get_sku(),
-			'permalink'         => $product->get_permalink(),
-			'images'            => ( new ProductImages() )->images_to_array( $product ),
-			'product_price'     => $this->prepare_money_response( $product->get_price(), wc_get_price_decimals() ),
```

Comment:
Note; I removed this intentionally as it isn't used. Cart should only need the line totals below.

Reply:
Documentation should be updated.

So is `line_subtotal` equivalent to what `product_price` was?

Reply:
Replied https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1394#discussion_r358787705

Readme is updated already. No usage of this yet.